components/gateway/internal/auditlog/service_test.go

```diff
@@ -221,6 +223,41 @@ func TestAuditlogService_LogConfigurationChange(t *testing.T) {
 	})
 
 }
+func TestSink_ChannelStuck(t *testing.T) {
+	//GIVEN
+	timeout := time.Millisecond * 100
```

Comment:
why this test is so complicated:
- sink.Log is synchronous function AFAIK, so you don't have to create goroutine
- a channel can be not buffered - then if you will send only one message and don't read from the channel you will get an error
- please check if we do not have a resource leak (if channel should not be closed at the end of the test)